.eslintrc.json

```diff
@@ -0,0 +1,28 @@
+{
+  "env": {
+    "browser": true,
+    "es2021": true
+  },
+  "extends": "eslint:recommended",
+  "parserOptions": {
+    "ecmaVersion": "latest",
+    "sourceType": "module"
+  },
+  "ignorePatterns": ["!lib/", "node_modules/"],
+  "overrides": [
+    {
+      "files": ["src/**/*.test.js", "**/*.cy.js"],
```

Comment:
Instead of merging these together, there should be one object in the array for cypress files and one for jest.

Reply:
Oliver, thank you for your feedback. Everything seems to be fixed now.